intranet/templates/bus/morning.html

```diff
@@ -0,0 +1,145 @@
+{% extends "page_with_nav.html" %}
```

Comment:
Could we reduce duplication between this and `intranet/templates/bus/home.html`?

Reply:
I cleaned up some of the uneeded script in `intranet/templates/bus/morning.html`, but there isn't that much similarity between the two in terms of the main content, i.e. neither naturally extends the other. Most of the scripts in `{% block js %}` are the same and of course the dark mode/css imports are as well, but everything else is different. Do you want me to create a parent page for them both to extend?

Reply:
The particular redundancies I see are:
* duplicate settings in the `js` block. I would put them in an `include`d tempate
* duplicate `enable_bus_driver` logi in the `js` block, I would move them to a common template.